### ISSUE TITLE
Did you reproduce the result in paper?

### ISSUE THREAD

**Issue by CoinCheung** [dongkyuk/DOLG-pytorch#1]
If I simply replace resnet101 with this, should I adjust learning rate?

**Comment by dongkyuk** [comment]
I couldn't reproduce the results in my machine because of training time. 
The paper used 512 images with batch size of 128 and had 8 V100 GPUs, which is uncomparable to mine.
I'm not sure how much batch size you can fit in your machine, but the learning rate should be tuned according to batch size. 
I recommend tuning the learning rate as described in this ![paper](https://arxiv.org/abs/1506.01186]). 
If you are using pytorch lightning, you can use their ![Learning Rate Finder](https://pytorch-lightning.readthedocs.io/en/latest/advanced/lr_finder.html)
